tests/Tests/300__managedservices_requirements/310__sre_signals_and_troubleshooting/logged-users.robot

```diff
@@ -0,0 +1,31 @@
+*** Settings ***
+Resource         ../../../Resources/ODS.robot
+Resource         ../../../Resources/Common.robot
+Resource         ../../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+Suite Setup      Begin Web Test
+Suite Teardown   End Web Test
+
+*** Test Cases ***
+Logout and Log Back in as an Admin
```

Comment:
Title should be the same or similar as in Polarion: `Verify "logged in users" are displayed in the dashboard`